src/test/java/net/smartcosmos/dto/metadata/MetadataCreateTest.java

```diff
@@ -0,0 +1,54 @@
+package net.smartcosmos.dto.metadata;
```

Comment:
Do we want to test every method for a model?

The only thing tested right now is the correct initialization of the version field, because default values can go missing with Lombok builders if not treated carefully.